tests/test_convertors.py

```diff
@@ -58,15 +59,30 @@ def test_datetime_convertor(test_client_factory: TestClientFactory, app: Router)
     )
 
 
-@pytest.mark.parametrize("param, status_code", [("1.0", 200), ("1-0", 404)])
-def test_default_float_convertor(test_client_factory: TestClientFactory, param: str, status_code: int) -> None:
-    def float_convertor(request: Request) -> JSONResponse:
+@pytest.mark.parametrize(
+    "param_type, param_value, status_code",
+    [
+        ("float", "1.0", 200),
+        ("float", "1-0", 404),
+        ("uuid", "00000000-aaaa-ffff-9999-000000000000", 200),
+        ("uuid", "00000000aaaaffff9999000000000000", 200),
+        ("uuid", "00000000-AAAA-FFFF-9999-000000000000", 200),
+        ("uuid", "00000000AAAAFFFF9999000000000000", 200),
+        ("uuid", "not-a-uuid", 404),
+    ],
+)
+def test_default_convertor(
+    test_client_factory: TestClientFactory, param_type: str, param_value: str, status_code: int
+) -> None:
+    expected_types = {"float": float, "uuid": UUID}
```

Comment:
Can we avoid this by leaving the `test_default_float_convertor` unchanged and create a new one for `uuid`? 🙏

Reply:
Done :heavy_check_mark: